kong/plugins/request-transformer/access.lua

```diff
@@ -303,7 +303,33 @@ local function transform_body(conf)
   end
 end
 
+local function transform_method(conf)
+  if conf.http_method then
+    ngx.req.set_method(ngx["HTTP_"..conf.http_method:upper()])
```

Comment:
All `ngx` methods are cached except this one. It should be too for consistency and efficiency.